lib/mayaUsdAPI/mayaUsdAPI.h.src

```diff
@@ -47,4 +47,11 @@ using namespace MAYAUSDAPI_VERSIONED_NS;
 #define MAYAUSDAPI_NS_DEF MAYAUSDAPI_VERSIONED_NS
 #endif
 
+// Any MayaUsd cmake defines that are used in the MayaUsdAPI project
+// must be added here so that clients using the API will have the
+// needed defines.
+#ifndef WANT_MATERIALX_BUILD
+#cmakedefine WANT_MATERIALX_BUILD
+#endif
```

Comment:
This will give the following when it is or is not defined. I needed to add the ifndef as clients like MayaHydra are setting this variable in their own cmake files.
```
#ifndef WANT_MATERIALX_BUILD
#define WANT_MATERIALX_BUILD
#endif

#ifndef WANT_MATERIALX_BUILD
/* #undef WANT_MATERIALX_BUILD */
#endif
```